odoo/addons/base/models/ir_ui_view.py

```diff
@@ -1277,7 +1279,8 @@ def distribute_branding(self, e, branding=None, parent_xpath='',
         if not e.get('data-oe-model'):
             return
 
-        if {'t-esc', 't-raw'}.intersection(e.attrib):
+        is_replace_wrapping_el = e.attrib.pop('meta-oe-xpath-wrapping', None)
+        if is_replace_wrapping_el or {'t-esc', 't-raw'}.intersection(e.attrib):
```

Comment:
Why not just add `'meta-oe-xpath-wrapping'` to the set? In fact why even set a value for this attribute? Just set the attribute with an empty value as a "marker"?

Reply:
Because I actually remove that marker from the node (so it is just there for the time of rendering but not served to the client), just like the `meta-oe-xpath-replacing` whose logic I copied. I can change that if you want of course.